charts/kargo/values.yaml

```diff
@@ -26,7 +26,12 @@ global:
   podLabels: {}
   ## @param global.podAnnotations Annotations to add to pods.
   podAnnotations: {}
-
+  ## Service Account global settings
+  serviceAccount:
```

Comment:
I appreciate you having done it this way. I regret that `labels` and `annotations` above were added without specifying _what_ those are applied to -- which led to the necessity of the ugly `podLabels` and `podAnnotations`. I'd love to change that if it weren't such a breaking change. At any rate, thank you for not perpetuating that ugly pattern.